packages/remix-dev/package.json

```diff
@@ -68,6 +68,19 @@
     "msw": "^0.39.2",
     "shelljs": "^0.8.5",
     "tiny-invariant": "^1.2.0",
-    "type-fest": "^2.12.2"
+    "type-fest": "^2.12.2",
+    "@yarnpkg/esbuild-plugin-pnp": "^2.0.0"
+  },
+  "peerDependencies": {
+    "@remix-run/serve": "1.5.1",
```

Comment:
Since we now have a remix package in `peerDependencies`, we'll need to update `updateRemixVersion` script util as well to account for it.

https://github.com/remix-run/remix/blob/5daa20d7992a061dedb6e8a1883fd259ae424ca9/scripts/utils.js#L138-L145